packages/@uppy/dashboard/src/components/FileItem/index.js

```diff
@@ -11,6 +11,20 @@ module.exports = class FileItem extends Component {
     return !shallowEqual(this.props, nextProps)
   }
 
+  componentDidMount () {
+    const file = this.props.file
+    if (!file.preview && this.props.handleRequestThumbnail) {
+      this.props.handleRequestThumbnail(file)
+    }
+  }
+
+  componentWillUnmount () {
+    const file = this.props.file
+    if (!file.preview && this.props.handleCancelThumbnail) {
```

Comment:
I think we don't need the `this.props.handleCancelThumbnail` check

Reply:
my thinking was that the thumbnail request and cancellation are not inherently required to render FileItems, even though we do always pass the handlers in. Maybe that's too defensive, I guess we can add this back if we do ever end up needing it :)